build_tools/python/benchmarks/iree/definitions.py

```diff
@@ -45,19 +63,22 @@ def generate(
             compile_config=cls.CASCADELAKE_COMPILE_CONFIG, model=model)
         for model in model_groups.MOBILE
     ]
+    # Generate compile specs for workstation models.
+    workstation_model_compile_specs = [
```

Comment:
Hmm "workstation" seems like an odd name here. "Server"? That's already super overloaded. "Desktop"? Also inaccurate. Hmmm... But generally I don't think the main use case for these is someone's workstation. Desktop seems the least bad because it's more common for this distinction, I think and less overloaded than server, even though it's totally inaccurate (who keeps a computer on top of a desk these days?). As a further out option, the thing that is actually the critical difference here is the specs of the machine/size of the model. "mobile" is just a proxy for low-power, low-cpu, etc. and "desktop/server/workstation" for high-power, high-cpu, etc. If someone put a big computer in a truck, we wouldn't use our "mobile" configs, and we're going to use our mobile configs on non-mobile edge devices (like a Google home or something). Perhaps a better nomenclature would be "high resource" and "low resource".

Reply:
Decided to simply use SMALL and LARGE to roughly classify them, which I guess is commonly used in ML for describing how computationally-heavy a model is.